cdap-metadata-ext-spanner/src/main/java/io/cdap/cdap/metadata/spanner/SpannerConfig.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.cdap.metadata.spanner;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Spanner;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Utility class for spanner metadata storage.
+ */
+class SpannerConfig {
+
+  private static final String PROJECT = "project";
+  private static final String INSTANCE = "instance";
+  private static final String DATABASE = "database";
+
+  static DatabaseAdminClient getSpannerDbAdminClient(Spanner spanner) {
+    return spanner.getDatabaseAdminClient();
+  }
+
+  static String getInstanceID(Map<String, String> cConf) {
+    String instance = cConf.get(INSTANCE);
+    return Objects.requireNonNull(instance);
+  }
+
+  static String getDatabaseID(Map<String, String> cConf) {
+    String database = cConf.get(DATABASE);
+    return Objects.requireNonNull(database);
+  }
+
+  static String getProjectID(Map<String, String> cConf) {
+    String project = cConf.get(PROJECT);
+    return Objects.requireNonNull(project);
+  }
```

Comment:
Mention access modifiers for the methods. I believe this is meant to be public and not internal?

Also add java docs for public methods.